### PR TITLE
Remove check_http and check_curl test which are somehow always failing

### LINKED ISSUE #1776: CI failing cause www.e-paycobalt.com
CI tests are [failing](https://github.com/monitoring-plugins/monitoring-plugins/runs/7336242150?check_suite_focus=true) with `Failed test 'Can read https for www.e-paycobalt.com (uses AES certificate)'`

### PR DESCRIPTION
**Pull request by RincewindsHat**
Resolves #1776

### CODE REVIEW

File: plugins/t/check_curl.t [hunk 1]
```diff
@@ -184,15 +184,6 @@ SKIP: {
                 is( $res->return_code, 2, "Certificate expired exit code" );
         };
 
-        $res = NPTest->testCmd( "./$plugin --ssl $host_tls_http -E" );
-        like  ( $res->output, '/time_connect=[\d\.]+/', 'Extended Performance Data Output OK' );
-        like  ( $res->output, '/time_ssl=[\d\.]+/', 'Extended Performance Data SSL Output OK' );
```
Comment (sni) [inline plugins/t/check_curl.t:189]: I think those should not be removed

File: plugins/t/check_http.t [hunk 2]
```diff
@@ -162,16 +162,6 @@ SKIP: {
                 like($res->output, qr/CRITICAL - Certificate '$host_tls_cert' expired on/, "Certificate expired output");
         };
 
-        $res = NPTest->testCmd( "./$plugin --ssl $host_tls_http -E" );
-        like  ( $res->output, '/time_connect=[\d\.]+/', 'Extended Performance Data Output OK' );
-        like  ( $res->output, '/time_ssl=[\d\.]+/', 'Extended Performance Data SSL Output OK' );
-
```
Comment (sni) [inline plugins/t/check_http.t:168]: same here.

**Review by sni — COMMENTED** [review]
It seems like you removed to many?

File: plugins/t/check_http.t (hunk 2 above)
Comment (RincewindsHat) [reply plugins/t/check_http.t:168]: indeed. I should not try to multi-task.

**Review by sni — APPROVED** [review]
looks good.

**Pull request merged**